_test/deploy-gatekeeper.sh

```diff
@@ -77,7 +77,6 @@ generate_constraints() {
   # TODO LIST
   # - ocp/bestpractices/container-resources-limits-memory-greater-than: TODO: https://github.com/redhat-cop/rego-policies/issues/76
   # - ocp/bestpractices/container-resources-requests-memory-greater-than: TODO: https://github.com/redhat-cop/rego-policies/issues/76
-  # - combine: TODO: https://github.com/redhat-cop/rego-policies/issues/70
   konstraint create --ignore "(podman|ocp\/deprecated|ocp\/bestpractices\/deploymentconfig-triggers-notset|ocp\/bestpractices\/rolebinding-roleref-apigroup-notset|ocp\/bestpractices\/rolebinding-roleref-kind-notset|ocp\/bestpractices\/container-resources-limits-memory-greater-than|ocp\/bestpractices\/container-resources-requests-memory-greater-than|combine)"
```

Comment:
`--ignore` doesn't exist in latest version `0.5.1 `😔

```
$ GO111MODULE=on go get github.com/plexsystems/konstraint
go: downloading github.com/plexsystems/konstraint v0.5.1
go: github.com/plexsystems/konstraint upgrade => v0.5.1

                                                                                                                                                                                      
$ _test/deploy-gatekeeper.sh deploy_constraints

Deleting all ConstraintTemplates...
No resources found
Creating ConstraintTemplates via konstraint...
Error: unknown flag: --ignore
Usage:
  konstraint create <dir> [flags]
                                                                                                                                                                                    
$ konstraint --help   
A tool to create and manage Gatekeeper CRDs from Rego

Usage:
  konstraint [command]

Available Commands:
  create      Create Gatekeeper constraints from Rego policies
  doc         Generate documentation from Rego policies
  help        Help about any command

Flags:
  -h, --help      help for konstraint
  -v, --version   version for konstraint

Use "konstraint [command] --help" for more information about a command.

Reply:
Yeah, i'd seen they've removed that flag but just not got round to fixing it. I'll see if i can switch it to using `find -exec rm`

Reply:
> @ckavili ; pushed an update for the `ignore`

I'd suggest creating constraint templates first then apply constraints with sleep checks because in my case, `find` listed a  `constraint.yaml` and tried to apply it without its template definition and got an error  🤦🏻‍♀️